tests/core/pyspec/eth2spec/utils/hash_function.py

```diff
@@ -6,4 +6,8 @@
 
 
 def hash(x: Union[bytes, bytearray, memoryview]) -> Bytes32:
-    return Bytes32(sha256(x).digest())
+    try:
```

Comment:
Did you run into an issue hashing something?

Reply:
Ah yes, lost track of some debug thing. It's the config that's hashable. Changed it to NamedTuple later (stricter validation, simple to embed as key for caches). TypedDict isn't strict enough. And a dataclass, even with its special decorator options, doesn't work: its generated `__hash__` calls the global `hash`, and it turned out to call the SSZ hash function, for the config data as a whole, which doesn't work.